### ISSUE TITLE
About the multi-task loss function

### ISSUE THREAD

**Issue by chengjianhong** [maudzung/TTNet-Realtime-for-Table-Tennis-Pytorch#18]
Hi, according to Eq.(6), the regularizer may be less than zero, and even the total loss is less than zero during the training. In my segmentation and classification tasks, this scenario is existing if using the homoscedastic uncertainty loss. But I can't understand whether it's normal or not?  How do you think about it?
